autogpt_platform/backend/backend/server/rest_api.py

```diff
@@ -635,11 +632,6 @@ def get_execution_schedules(
         execution_scheduler = self.execution_scheduler_client
         return execution_scheduler.get_execution_schedules(graph_id, user_id)
 
-    @expose
```

Comment:
I moved this to DatabaseAPI, so that executor manager no longer communicate with the agent server API.

Also it doesn't make sense to make agent server a pyro service, we can make this rest_api a normal fast API process now. cc @Swiftyos

Reply:
Thats amazing. Thank you :)